docs/Project-Configuration/Project-Commands.md

```diff
@@ -2,6 +2,26 @@
 
 Project Commands are the fundamental units of functionality in an Evergreen task.
 
+# Basic Command Structure
+
+```yaml
+- command: shell.exec
+  display_name: run my cool script ## optional 
+  type: system ## optional
+  timeout_secs: 10 ## optional
+  params:
+    script: echo "my script"
+```
+Explanation:
+
+-   `command`: a command name predefined by evergreen
+-   `display_name`: an optional user defined display name for the command. This will show up in logs and in the UI
+     with more details, for example: 'shell.exec' ('run my cool script') (step 1 of 1)
+-   `type`: an optional command type. This will affect the [failure colors](../Project-Configuration/Project-Configuration-Files.md#command-failure-colors)
```

Comment:
I don't think we need `../ProjectConfiguration` since it's already within that directory, no? Since it's at the same level of the hierarchy as the project configuration file docs, just referencing `Project-Configuration-Files.md#command-failure-colors` should be enough.